<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scurve jogging</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(51, 57, 59);
color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="3" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="4" column="1">
          <widget class="QLabel" name="label_vel">
           <property name="text">
            <string>0.000</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label_pos">
           <property name="text">
            <string>0.000</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_acc">
           <property name="text">
            <string>0.000</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>acceleration </string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>[mm/s]</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>[s]</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>velocity</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>time</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>output:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>[mm/s2]</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_time">
           <property name="text">
            <string>0.000</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>[mm]</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>position</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>period</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>[1-2-3-4-5-6-7-8-10]</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_period">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_jog_x_min">
           <property name="text">
            <string>x -</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_jog_x_plus">
           <property name="text">
            <string>x +</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>controls:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_reset">
           <property name="text">
            <string>reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>velocity max </string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>[mm/s]</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineEdit_acc_max">
           <property name="text">
            <string>20</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLineEdit" name="lineEdit_acc_interval">
           <property name="text">
            <string>0.020</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="lineEdit_vel_max">
           <property name="text">
            <string>60</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>acceleration interval</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>input:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>[mm/s2]</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>acceleration max </string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>time interval</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>[s]</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="lineEdit_interval_time">
           <property name="text">
            <string>0.001</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>[a]</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_vel_max_mm_min">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>[mm/min]</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>coded by skynet cyberdyne 12-2022</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_opengl"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
